### DISCUSSION TITLE
AUTH:INDI

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#391] [Q&A] [answered]
I am trying to make my tree compatible to Gedcom 7.
What is the best way to refer to a person, who is the author/initiator of something:
0 @F1@ FAM
1 MARR
2 HUSB @I2@
2 WIFE @I3@
2 SOUR @S4@
...
0 @S4@ SOUR
1 AUTH
2 INDI @I5@

But AUTH:INDI is not invalid.

Hope you can help me.

Marianne van Harten

**Comment by tychonievich** [comment, marked as answer]
This is not directly supported in the current spec. However, it is very doable using extensions. The `ALIA` structure is the closest match to this use case, so I'd recommend using a relocated extension strcuture, like so:

Add to the HEAD the following

```gedcom
1 SCHMA
2 TAG _ALIA https://gedcom.io/terms/v7/ALIA
```

and then use it as

```gedcom
0 https://github.com/s4@ SOUR
1 AUTH
2 _ALIA @i5@
```

You are also welcome to create your own extension if you don't like relocating ALIA.
